packages/react/src/image.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "@hpcc-js/preact-shim";
+
+interface Image {
+    href: string;
+    x?: number;
+    y?: number;
+    height?: number;
+    yOffset?: number;
+}
+
+export const Image: React.FunctionComponent<Image> = ({
+    href,
+    x,
+    y = 0,
+    height = 12
+}) => {
+
+    return <image
```

Comment:
Should this just be an `<img... />` or `<picture>`? Or is there a library somewhere defining that <image> element?

Reply:
Its one of these:  https://developer.mozilla.org/en-US/docs/Web/SVG/Element/image